lib/workers/global/config/parse/file.ts

```diff
@@ -71,6 +71,13 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
       logger.debug('No config file found on disk - skipping');
     }
   }
+
+  // Avoid deleting the default config
+  // Attempt deletion only if RENOVATE_CONFIG_FILE is specified
+  if (env.RENOVATE_CONFIG_FILE) {
```

Comment:
i would move this to the function too, so we've less nesting and less conditions